users/serializers.py

```diff
@@ -29,6 +34,11 @@
     """,
     flags=re.I | re.VERBOSE | re.MULTILINE,
 )
+NAME_CONTAINS_HTML_URL_MSG = (
+    "Full name can not contain HTML or URL. Please try a different one."
+)
+
+OPENEDX_NAME_VALIDATION_MSGS_MAP = {"Enter a valid name": NAME_CONTAINS_HTML_URL_MSG}
```

Comment:
It is not always guaranteed that the incorrectness of the error message will be based on the HTML or URL.